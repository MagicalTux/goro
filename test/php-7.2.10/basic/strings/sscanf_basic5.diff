     *** Testing sscanf() : basic functionality -using exponential format ***
     
     -- Try sccanf() WITHOUT optional args --
004- float(10.12345)
005- float(10123.45)
006- float(10123.45)
007- float(-101234.5)
008- float(10.12345)
009- float(10123.45)
010- float(10123.45)
011- float(-101234.5)
     
013- -- Try sccanf() WITH optional args --
014- int(4)
015- float(10.12345)
016- float(10123.45)
017- float(10123.45)
018- float(-101234.5)
019- int(4)
020- float(10.12345)
021- float(10123.45)
022- float(10123.45)
023- float(-101234.5)
024- ===DONE===
005+ Fatal error: Uncaught Error: Call to undefined function sscanf in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/sscanf_basic5.php:16
006+ Stack trace:
007+ #0 {main}
008+   thrown in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/sscanf_basic5.php on line 16
