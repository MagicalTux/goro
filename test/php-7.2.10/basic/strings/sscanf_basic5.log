
---- EXPECTED OUTPUT
*** Testing sscanf() : basic functionality -using exponential format ***

-- Try sccanf() WITHOUT optional args --
float(10.12345)
float(10123.45)
float(10123.45)
float(-101234.5)
float(10.12345)
float(10123.45)
float(10123.45)
float(-101234.5)

-- Try sccanf() WITH optional args --
int(4)
float(10.12345)
float(10123.45)
float(10123.45)
float(-101234.5)
int(4)
float(10.12345)
float(10123.45)
float(10123.45)
float(-101234.5)
===DONE===
---- ACTUAL OUTPUT
*** Testing sscanf() : basic functionality -using exponential format ***

-- Try sccanf() WITHOUT optional args --

Fatal error: Uncaught Error: Call to undefined function sscanf in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/sscanf_basic5.php:16
Stack trace:
#0 {main}
  thrown in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/sscanf_basic5.php on line 16
---- FAILED
