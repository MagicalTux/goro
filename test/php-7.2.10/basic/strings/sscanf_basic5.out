*** Testing sscanf() : basic functionality -using exponential format ***

-- Try sccanf() WITHOUT optional args --

Fatal error: Uncaught Error: Call to undefined function sscanf in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/sscanf_basic5.php:16
Stack trace:
#0 {main}
  thrown in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/sscanf_basic5.php on line 16